MimeKit/Cryptography/Ed25519DigestSigner.cs

```diff
@@ -69,6 +72,15 @@ public void BlockUpdate (byte[] input, int inOff, int length)
 			digest.BlockUpdate (input, inOff, length);
 		}
 
+#if NETCOREAPP2_1_OR_GREATER || NETSTANDARD2_1_OR_GREATER
+		/// <summary>Update the signer with a span of bytes.</summary>
+		/// <param name="input">the span containing the data.</param>
+		public void BlockUpdate (ReadOnlySpan<byte> input)
+		{
+			digest.BlockUpdate (input);
```

Comment:
TODO: ??

Reply:
Drop NETCOREAPP2_1_OR_GREATER - MimeKit doesn't target NetCoreApp2.1+.

What is there left TODO?

Reply:
ISigner requires:
```csharp
#if NETCOREAPP2_1_OR_GREATER || NETSTANDARD2_1_OR_GREATER
        /// <summary>Update the signer with a span of bytes.</summary>
        /// <param name="input">the span containing the data.</param>
        void BlockUpdate(ReadOnlySpan<byte> input);
#endif
```

I can change `NETCOREAPP2_1_OR_GREATER ` to  `NET6_0_OR_GREATER` if you want.

Reply:
The TODO is for the compilation errors: 

Error	CS7036	There is no argument given that corresponds to the required parameter 'inOff' of 'IDigest.BlockUpdate(byte[], int, int)'